### DISCUSSION TITLE
Printer Option in Stable Version 0.5.2?

### DISCUSSION THREAD

**Discussion by Biniou671** [flybywiresim/aircraft#3499] [General]
Hi Everyone,
Just wondering if it could be possible, just to add the printer option in the stable version 0.5.2?
This printer option is so useful.

Thanks again to FBW team for their amazing job.

**Comment by hotshotp** [comment]
No, stable will be released when there are enough features. It will probably come soon because there has not been a new stable for a while

**Reply by Biniou671** [reply]
Noted with thanks.

**Comment by robinottawa** [comment]
What is a printer option?

**Reply by sidnov** [reply]
The printer located behind the TCAS panel allows you to print small sheets of paper with METAR info as of now, via the MCDU

**Reply by cadilhac** [reply]
and TAF and ATIS if I'm right.

**Comment by tocdoc** [comment]
I´ve tried the printer in the experimental version without any problem. So I wonder why it is not yet included in the stable version.

**Reply by hotshotp** [reply]
because stable is released after all the features are ready and it will be released in 0.6.0

**Comment by TLSY2** [comment]
hmm, use dev in my opinion, i find it same as stable just more features and it gets updated like daily
